### ISSUE TITLE
Non-string arguments in parametrized query throw error

### ISSUE THREAD

**Issue by bartlomieju** [bartlomieju/deno-postgres#9]
```ts
const result = await client.query('SELECT * FROM ids WHERE id < $1;', 2); // notice number here
```

results in:
```
TypeError: input is not iterable
    at stringToCodePoints (gen/bundle/main.js:4438:23)
    at TextEncoder.encode (gen/bundle/main.js:4664:42)
    at config.args.forEach.arg (/dev/deno-postgres/connection.ts:239:49)
    at Array.forEach (<anonymous>)
    at Connection._sendBindMessage (/dev/deno-postgres/connection.ts:230:21)
    at Connection._preparedQuery (/dev/deno-postgres/connection.ts:315:20)
```

**Comment by bartlomieju** [comment]
Fixed in `f020b52fbdc0efe1c8529a6db65d04f31295bf24`, still requires tests

**Comment by bartlomieju** [comment]
Closed in #19
